index.js

```diff
@@ -4,7 +4,7 @@ const requestBuilder = require('./requestBuilder');
 const Parsers = require('./parsers');
 
 class Darwin {
-  constructor(apiKey, options) {
+  constructor(apiKey, options = {}) {
```

Comment:
Some of these were failing if there were no options objects, because it was trying to assign to a missing object. I went ahead and added a default object to them all